### ISSUE TITLE
Feature Request: possible to add filenames?

### ISSUE THREAD

**Issue by koaning** [nschloe/deadlink#30]
I like this project! 

I'd like to use it as a CI step for my [calmcode](https://calmcode.io/) project, but since that project contains hundreds of pages that it needs to check I was wondering if it's possible to add a flag such that your tool could also tell me what file the link was found in. I'll gladly hear it though if this is out of scope.

**Comment by nschloe** [comment]
You mean like grep?

**Comment by koaning** [comment]
I didn't know grep could be used to search through multiple files. But if so, I suppose so.

**Comment by koaning** [comment]
It's mainly that if my project has 1000 pages with dead links, it's great to know that there's a deadlink ... but in order to fix it I'd need to know which page to fix. By adding the filename where the deadlink was found the process would be less manual.

**Comment by nschloe** [comment]
In the output of `deadlink check`, yes? That'd probably make some sense.

**Comment by koaning** [comment]
> In the output of deadlink check, yes? That'd probably make some sense.

Correct! I suppose it wouldn't hurt to have a flag for it, but I'll gladly leave it up to you if that makes sense.

**Comment by Prof-Florian-Kuenzner** [comment]
Hello, for me, the possibility to output the `filename` and the `line number` would be very helpful.
